### PR TITLE
Configure jpackage 

### PR DESCRIPTION
**Pull request by aalmiray**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -210,14 +210,28 @@ configure(project(':app:fx')) {
             // start scripts created by the application plugin are overwritten by the
             // org.beryx.runtime plugin, thus custom settings must be configured here as
             // opposed to in the usual `startScripts` closure
-            jvmArgs = [ '--add-modules=javafx.controls' ]
```
Comment (cbeams) [inline build.gradle:213]: @aalmiray is there a reason this has to be removed? If it is, `bisqfx` no longer runs on my M1 mac. Fails with 
'No Tookit fourd" exception.

Comment (aalmiray) [reply build.gradle:213]: Yes, it's not needed for OSX-intel. What's the output you get from `${osdetector.arch} when run on M1?

Comment (cbeams) [reply build.gradle:213]: `osdetector.arch == aarch_64`

If it is merely not needed on macos intel, but does not cause any problems, I would just leave it as-is. If it blows up on macos intel, then, yeah, let's put a conditional in place to include it only on `aarch_64`.

Comment (aalmiray) [reply build.gradle:213]: Fixed.

**Pull request merged**